ebuild-writing/misc-files/patches/text.xml

```diff
@@ -112,6 +112,42 @@ from the <c>vim</c> patch tarball:
 </body>
 </section>
 
+<section>
+<title>Conditional Patching</title>
+<body>
+<p>
+Patching is ideally only done to make the package in question build properly,
+and should not be done to modify the runtime behaviour of the package. This is
+what USE flags and features of the package are for. As such, it is preferable to
+apply patches unconditionally and avoid conditional patching.
+</p>
+<p>
+Consider the following example <c>src_prepare()</c> implementation:
+</p>
+<codesample lang="ebuild">
+src_prepare() {
+	if use threads; then
+		PATCHES+=( "${FILESDIR}/${P}"-mt.patch )
```

Comment:
The quoting looks a little strange to my eyes. Either quote only the variable where quoting is necessary:
```suggestion
		PATCHES+=( "${FILESDIR}"/${P}-mt.patch )
```
or quote the whole string:
```suggestion
		PATCHES+=( "${FILESDIR}/${P}-mt.patch" )
```

Reply:
I went with the second proposal - did you have a preference?

Reply:
Slight preference for `"${FILESDIR}"/${P}-mt.patch` here, but you'll find both styles in the tree.